io.github.unrud.RecentFilter.json

```diff
@@ -0,0 +1,46 @@
+{
+    "app-id": "io.github.unrud.RecentFilter",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "44",
+    "sdk": "org.gnome.Sdk",
+    "command": "recent-filter",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=xdg-data",
```

Comment:
this permission is not allowed by default

Reply:
Yes, I created a PR for an exception. The permission is required.

Reply:
Exception got merged.